book-store-ingress.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata: 
+  name: book-store-ingress
+  namespace: book-store-ns
+spec:
+  rules:
+  - host: book-store.shahargn.com
+    http:
+      paths: 
+      - path: /
+        pathType: Prefix 
+        backend:
+          service:
+            name: book-store-internal-service
+            port:
+              number: 8000
```

Comment:
Change it to `80` because you want to just been able to use the domain without specify a port in Insomnia, and the default port for `http` is `80`.

Reply:
the Ingress is by default open on port 80, he transfer the request to the service on port 8000 which his transfer to the pod also on port 8000. so we dont need to change the service to port 80 too. look at my screenshot

![image](https://user-images.githubusercontent.com/55717508/97780363-dce07100-1b8c-11eb-8c94-0d390fa43723.png)